Documentation/PLUGIN.md

```diff
@@ -182,6 +182,22 @@ exposed via public API, so even if `ImplementationOnlyImports` is set to `true`,
 this will only work if the `Visibility` is set to `internal`.
```

Comment:
Can we please add a note to both READMEs explaining how `ImplementationOnlyImports` interacts with `UseAccessLevelOnImports` (i.e. both can't be set/`internal` visibility for imports should be preferred over `@_implementationOnly`)?